app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
In case you are using annotation in other methods, this one should not be an exclusion. You can write signature like `def go_to_cafe(friends: list, cafe: Cafe) -> str:`